CHANGELOG.next.asciidoc

```diff
@@ -48,6 +48,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...main[Check the HEAD dif
 - Fix AWS config initialization issue when using a role {issue}30999[30999] {pull}31014[31014]
 - Fix group write permissions on runtime directories. {pull}30869[30869]
 - Store syslog version as string. {pull}31446[31446]
+- Fix environment flag in Dockerfile CMD. {pull}31500[31550]
```

Comment:
That should go into `CHANGELOG.development.next.asciidoc` in the "added" section because it's not a user-facing change, it's just a documentation and internals update.

This is also not a fix because `-environment` and `--environment` behave the same way.

Reply:
Thanks for the PR review. I agree with your comment.

1. I moved the line CHANGELOG.next.asciidoc "fixed" to CHANGELOG-development.next.asciidoc "added"
2. I changed PR title and change log to `Remove ambiguity on Dockerfile CMD.`, to avoid the word "fix".